iped-carvers/iped-carvers-impl/src/main/java/iped/carvers/standard/AbstractCarver.java

```diff
@@ -150,7 +149,23 @@ public IItem carveFromFooter(IItem parentEvidence, Hit footer) throws IOExceptio
 
         return null;
     }
+    
```

Comment:
Good new methods to customize carver. 
But wouldn't it be interesting to specify them in Carver interface?

Reply:
Maybe is better a new issue. And these prefix and suffix could also be allowed to be configured in CarverConfig.